Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -326,6 +375,23 @@ async def test_module(base_url: str, client_id: str, client_secret: str, verify_
         demisto.results('ok')
 
 
+def get_sample_events() -> None:
+    """Extracts sample events stored in the integration context and returns them
+
+    Returns:
+        None: No data returned.
+    """
+    integration_context = demisto.getIntegrationContext()
+    sample_events = integration_context.get('sample_events')
+    if sample_events:
+        try:
+            demisto.results(json.loads(sample_events))
+        except json.decoder.JSONDecodeError as e:
+            raise ValueError(f'Failed deserializing sample events - {e}')
+    else:
+        demisto.results('No sample events found.')
```

Comment:
Let's also check if `store_samples` is enabled. If not, specify in the message that you must enable: `store_samples` for this command to return results.